src/NUnitFramework/tests/WorkItemTests.cs

```diff
@@ -171,7 +171,7 @@ class FakeTestResult : TestResult
             public FakeTestResult(ITest test) : base(test)
             {
             }
-
+            public override int TotalCount => 1;
```

Comment:
Can we insert an empty line before and after this line to keep the file consistent.

Reply:
Done.